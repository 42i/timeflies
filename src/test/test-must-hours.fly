day 2012-01-01
must-hours tue=8.0 wed=8.0 thu=8.0 fri=8.0
day 2012-01-08
must-hours mon=8.0 tue=8.0 wed=8.0 thu=4.0 fri=4.0
day 2012-01-15
must-hours mon=8.0 tue=8.0 wed=8.0 thu=8.0 fri=8.0
day 2012-01-22
must-hours mon=2.0 tue=3.0 wed=4.0 thu=4.5 fri=5.5
leave-days 2012-01-01 2012-01-07; first
leave-days 2012-01-08 2012-01-14; second
leave-days 2012-01-15 2012-01-21; third
sick-days 2012-01-23 2012-01-28; sicko