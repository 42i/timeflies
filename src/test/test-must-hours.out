Time at work overview (all):
     when        worked   leave    sick balance
  week 2011-52: ----.-- ----.-- ----.-- ----.--
 month 2011-01: ----.-- ----.-- ----.-- ----.--
2012-01-03 Tue: ----.--    8.00 ----.-- ----.-- first
2012-01-04 Wed: ----.--    8.00 ----.-- ----.-- first
2012-01-05 Thu: ----.--    8.00 ----.-- ----.-- first
2012-01-06 Fri: ----.--    8.00 ----.-- ----.-- first
  week 2012-01: ----.--   32.00 ----.-- ----.--
2012-01-09 Mon: ----.--    8.00 ----.-- ----.-- second
2012-01-10 Tue: ----.--    8.00 ----.-- ----.-- second
2012-01-11 Wed: ----.--    8.00 ----.-- ----.-- second
2012-01-12 Thu: ----.--    4.00 ----.-- ----.-- second
2012-01-13 Fri: ----.--    4.00 ----.-- ----.-- second
  week 2012-02: ----.--   32.00 ----.-- ----.--
2012-01-16 Mon: ----.--    8.00 ----.-- ----.-- third
2012-01-17 Tue: ----.--    8.00 ----.-- ----.-- third
2012-01-18 Wed: ----.--    8.00 ----.-- ----.-- third
2012-01-19 Thu: ----.--    8.00 ----.-- ----.-- third
2012-01-20 Fri: ----.--    8.00 ----.-- ----.-- third
  week 2012-03: ----.--   40.00 ----.-- ----.--
2012-01-23 Mon: ----.-- ----.--    2.00 ----.-- sicko
2012-01-24 Tue: ----.-- ----.--    3.00 ----.-- sicko
2012-01-25 Wed: ----.-- ----.--    4.00 ----.-- sicko
2012-01-26 Thu: ----.-- ----.--    4.50 ----.-- sicko
2012-01-27 Fri: ----.-- ----.--    5.50 ----.-- sicko
  week 2012-04: ----.-- ----.--   19.00 ----.--
 month 2012-01: ----.--  104.00   19.00 ----.--
         total: ----.--  104.00   19.00 ----.--
     when        worked   leave    sick balance
